tests/integration/cocoon/test_cocoon_instruments.py

```diff
@@ -782,3 +782,88 @@ def test_load_from_data_frame_instruments_with_strip(
                 for key, value in expected_outcome["instruments"].values.items()
             )
         )
+
+    def test_load_instrument_properties(
+        self,
+    ):
+        data_frame = pd.DataFrame({
+            "instrument_name": ["BP_LondonStockEx_BP", "Glencore_LondonStockEx_GLEN", "TESCO_LondonStockEx_TSCO"],
+            "figi": ["BBG000C05BD1", "BBG001MM1KV4", "BBG000BF46Y8"],
+            "isin": ["BBG000C05BD1", "JE00B4T3BW64", "GB0008847096"],
+            "client_internal": ["imd_43535553", "imd_34534555", "imd_34634673"]
+        })
+
+        scope = "TestScope1"
+        mapping_required = {"name": "instrument_name"}
+        identifier_mapping = {"Figi": "figi", "Isin": "isin", "ClientInternal": "client_internal"}
+
+        # set up initial instruments
+        responses = cocoon.cocoon.load_from_data_frame(
+            api_factory=self.api_factory,
+            scope=scope,
+            data_frame=data_frame,
+            mapping_required=mapping_required,
+            mapping_optional={},
+            file_type="instruments",
+            identifier_mapping=identifier_mapping
+        )
+
+        self.assertGreater(len(responses["instruments"]["success"]), 0)
+
+        properties_df = pd.DataFrame({
+            "isin": ["BBG000C05BD1", "GG00B4L84979", "GB0031509804"],
+            "category": ["Oil & Gas", "Mining", "Retail"]
+        })
+
+        properties_required_mapping = {
+            "identifier": "isin",
+            "identifier_type": "$Isin"
+        }
+
+        result = cocoon.cocoon.load_from_data_frame(
+            api_factory=self.api_factory,
+            scope=scope,
+            data_frame=properties_df,
+            mapping_required=properties_required_mapping,
+            mapping_optional={},
+            file_type="instrument_property",
+            properties_scope=scope,
+            property_columns=["category"],
+            batch_size=1
+        )
+
+        errors = [success for batch in result["instrument_propertys"]["errors"] for success in batch]
+        successes = [success for batch in result["instrument_propertys"]["success"] for success in batch]
```

Comment:
Can you please rename the error variable to be error instead of success as per below?